utils/usePopper.ts

```diff
@@ -48,6 +61,8 @@ export function usePopper(options) {
               const { placement } = data;
 
               setState({ data, placement });
+
+              return data;
```

Comment:
This function appears to be a Popper.js [`ModifierFn`](https://github.com/FezVrasta/popper.js/blob/master/packages/popper/index.d.ts#L36) function which returns `data`:

```ts
export type ModifierFn = (data: Data, options: Object) => Data;
```

TypeScript warns with the following message if `data` is not returned:

```text
🛑 Types of property 'fn' are incompatible.
        Type '(data: Data) => void' is not assignable to type 'ModifierFn'.
          Type 'void' is not assignable to type 'Data'.ts(2345)
```